source/documentation/standards/how-to-get-a-domain-name.html.md.erb

```diff
@@ -0,0 +1,53 @@
+---
+title: How to get, register or manage a domain name
+last_reviewed_on: 2021-09-16
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+The MOJ centrally registers and maintains domains for use across the department.
+Domains used within the MOJ must follow the [naming domains]
+(naming-domains.html) standards and must be registered by the [MOJ's Domains
+team](mailto:domains@digital.justice.gov.uk).
+
+As per the [Naming domains guidance]
+(naming-domains.html#non-gov-uk-domains), there are various reasons why you
+shouldn't use a non-`gov.uk` domain name for your service.
+
+## Requesting a new domain or changes to an existing domain
+
+If you need to request a new domain or changes to an existing domain name, email
+the [MOJ's Domains team](mailto:domains@digital.justice.gov.uk). The Domains
+team will help ensure that your domain meets the [naming domains standard]
+(naming-domains.html) and our [wider standards for naming things]
+(naming-things.html). The Domains team will also help identify which subdomain
+is most appropriate for your service.
+
+You **must not** register a domain outside of the MOJ Domains team.
+
+## Moving your domain to the central register
+
+If you currently use or have registered a non-`gov.uk` domain, you **must**
+transfer the domain to the MOJ's Domains team. The Domains team will:
+
+- provide you with a new domain that meets [naming domains]
+  (naming-domains.html) standards
+- redirect your old domain to your new domain
+- help you decommission your old domain
+
+## Defensive domain registrations
+
+Defensive domains must be requested through the [MOJ's Domains team]
```

Comment:
Is it worth linking to some guidance here on what a defensive domain is and why you might want to register something in that way?

Reply:
Yep, security team are putting something together for this to link off to.

Reply:
Excellent.